web_no_crawler/__manifest__.py

```diff
@@ -1,17 +1,15 @@
-# -*- coding: utf-8 -*-
 # Copyright (C) 2018 Ventor, Xpansa Group (<https://ventor.tech/>)
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 
 {
     'name': 'Disallow indexing completely via robots.txt',
     'author': 'Ventor, Xpansa Group, Odoo Community Association (OCA)',
-    'website': 'https://github.com/OCA/website/tree/8.0',
+    'website': 'https://github.com/OCA/web',
     'category': 'web',
-    'version': '8.0.1.0.0',
+    'version': '12.0.1.0.0',
     'depends': [
-        'base',
+        'web',
     ],
     'installable': True,
-    'application': False,
-    'license': 'AGPL-3',
+    'license': 'LGPL-3',
```

Comment:
Why did you changed the license ?

Reply:
It was LGPL in all the licence statements. This is the odd one out.

Reply:
Thanks.